@types/swagger-client/schema.d.ts

```diff
@@ -3,23 +3,61 @@ declare module 'swagger-client/schema' {
 
   interface OpenAPIObject {
     paths: PathsObject;
+    components?: ComponentsObject;
+    security?: SecurityRequirementObject[];
   }
 
-  export interface PathsObject { [path: string]: PathItemObject }
-  export interface OperationObject {
-    operationId: string;
-    parameters: ParameterObject[];
-    responses: { [responseStatus: string]: ResponseObject };
+  interface ServerObject {
+    url: string;
+    description?: string;
+    variables?: {
+      [variableName: string]: ServerVariableObject;
+    };
   }
+
+  type NonEmptyArray<T> = [T, ...T[]];
+
+  interface ServerVariableObject {
+    enum?: NonEmptyArray<string>;
```

Comment:
What's this for?

Reply:
Swagger spec says if it exists it shouldn't be empty: https://swagger.io/specification/#server-variable-object
![image](https://user-images.githubusercontent.com/22045453/119551075-d3a03700-bd5e-11eb-9542-08408c3bbe65.png)